db/migrations/000003_add_order_settlements_table.up.sql

```diff
@@ -8,6 +8,7 @@ CREATE TABLE IF NOT EXISTS order_settlements (
     destination_chain_id TEXT NOT NULL,
     source_chain_gateway_contract_address TEXT NOT NULL,
     amount TEXT NOT NULL,
+    fee TEXT NOT NULL,
```

Comment:
imo we should keep the name of this value being `profit`, all of the existing references to this value call it (net) profit